relnotes

```diff
@@ -222,8 +222,7 @@ create_body () {
   echo
   echo "# $title"
   echo
-  echo "[Documentation](https://docs.k8s.io) &" \
-       "[Examples](https://releases.k8s.io/$CURRENT_BRANCH/examples)"
+  echo "[Documentation](https://docs.k8s.io)"
```

Comment:
@tpepper this hunk is the only change that applies to `release-notify`, run by anago here:
https://github.com/kubernetes/release/blob/920dcfb1bb3944c381e00c0701522263a2f419b6/anago#L445-L449

The *.go changes below are not (yet) used by `release-notify`, though I did test them via `go test`.

Reply:
looks safe to me, at least.

Reply:
Just to clarify, anago runs the command above, which drops the relnotes into the given `--release-bucket`.  The `release-notify` script downloads these notes when composing the email: 
https://github.com/kubernetes/release/blob/920dcfb1bb3944c381e00c0701522263a2f419b6/lib/releaselib.sh#L1042-L1052